viewer/packages/utilities/src/events/InputHandler.ts

```diff
@@ -0,0 +1,133 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+import { clickOrTouchEventOffset } from './clickOrTouchEventOffset';
+import { EventTrigger } from './EventTrigger';
+import debounce from 'lodash/debounce';
+import { assertNever } from '@reveal/utilities';
+
+type PointerEventDelegate = (event: { offsetX: number; offsetY: number }) => void;
+
+export class InputHandler {
+  private readonly domElement: HTMLElement;
+
+  private readonly _events = {
+    click: new EventTrigger<PointerEventDelegate>(),
+    hover: new EventTrigger<PointerEventDelegate>()
+  };
+
+  private setupEventListeners() {
```

Comment:
Private methods should be declared last.

Reply:
Move private functions to the after all public functions